### ISSUE TITLE
EMA support for CARD 

### ISSUE THREAD

**Issue by wrkhard** [lightning-uq-box/lightning-uq-box#186]

**Comment by nilsleh** [comment]
@wrkhard Thanks for the issue. Yeah ema support is a good idea. With diffusion training I have also noticed that usually, smaller learning rates, and gradient clipping are quiet beneficial. Also gradient accumulation depending on the batch size. The later two can easily be configured in the lightning `Trainer`  for example  `accumulate_grad_batches=4`, and `gradient_clip_val= 1.0`, in case you haven't tried that out.

**Comment by wrkhard** [comment]
@nilsleh thanks for the tips, I'll try those out!
